index.html

```diff
@@ -0,0 +1,137 @@
+    <title>cbradshaw-portfolio</title>
+</head>
+<body>
+<header>
+    <h1 class="name">Chris Bradshaw's Portfolio</h1>
+    <ul class="nav-links">
+        <li><a href="#about-me">About me</a></li>
+        <li><a href="#my-work">My work</a></li>
+        <li><a href="#contact">Email</a></li>
+    </ul>
+</header>
+<section id="about-me" class="banner">
+    <div class="portfolio-img">
+        <img id="profile-pic" src="./assets/images/new1.jpg" />
+    </div>
+    <div class="right">
+        <P>Maths teacher of 13 years looking to learn something new. Always had an interest in coding since writing bootdisks for games on my 486!
+        </P>
+        <div class="media-links">
+            <a id="link-button" href="https://www.linkedin.com/in/chris-bradshaw-39549421a/">
+                <img src="./assets/images/linked-in.png" alt="linked in logo" />
+            </a>
+            <a id="link-button" href="https://github.com/gingus55">
+                <img src="./assets/images/github.png" alt="github logo" />
+            </a>
+        </div>
+    </div>
+</section>
+<main class="main-container" id="my-work">
+<aside>
+    <ul class="aside-container">
+        <li>
+            <h2>Toolkit:</h2>
+        </li>
+        <li>
+            <h3>GIT</h3>
+            <img src="./assets/images/git-img.png" alt="git logo" />
+        </li>
+        <li>
+            <h3>HTML</h3>
+            <img src="./assets/images/html-logo-2.png" alt="html logo" />
+        </li>
+        <li>
+            <h3>CSS</h3>
+            <img src="assets/images/css-img.png" alt="css logo" />
+        </li>
+    </ul>
+</aside>
+<section class="project-container">
+    <div class="project-card primary">
+        <img src="./assets/images/horiseon-hpage.png" alt="webpage home screen-people around a laptop "  />
+        <div class="card-container">
+            <h4><b>Horiseon Project</b></h4> 
+            <p>Code refactored for clients needs.</p>
+            <a href="https://gingus55.github.io/marketing-accessibility/">
+                <h4>link to</h4>
+            </a>   
+        </div>
+    </div>
+    <div class="secondary-project-container">
+        <div class="project-card general">
+            <img src="https://via.placeholder.com/150" alt="placeholder image" />
+            <div class="card-container">
+                <h4><b>Coming soon</b></h4> 
+                <p>.................</p>
+                <a href="">
+                    <h4>link to</h4>
+                </a>   
+            </div>
+        </div>
+        <div class="project-card general">
+            <img src="https://via.placeholder.com/150" alt="placeholder image" />
+            <div class="card-container">
+                <h4><b>Coming soon</b></h4> 
+                <p>.................</p>
+                <a href="">
+                    <h4>link to</h4>
+                </a>    
+            </div>
+        </div><div class="project-card general">
```

Comment:
Start a new line here, you have done this a few times throughout the HTML

Reply:
I will look at spacing throughout my html. Thanks